src/Demos/HSP.lagda

```diff
@@ -268,65 +222,53 @@ f ⟨∘⟩ g = record  { f = (_⟨$⟩_ f) ∘ (_⟨$⟩_ g)
 \end{code}
 \fi
 
-\paragraph*{Inverses of setoid functions}
-We begin by defining an inductive type that represents the \emph{image} of a function.
-%\footnote{cf.~the \ualmodule{Overture.Setoid.Inverses} module of the \agdaalgebras library.}
+\paragraph*{Inverses}
+We frequently need to deal with inverses. These are most easily defined via the
```

Comment:
In the first sentence "inverses" seems a bit ambiguous (until you read on).

Actually, I don't think the phrase "image of a function" is technically correct.  I would prefer the following:

"We frequently need to deal with the inverse of a function. This is defined in terms of the image of the function's domain."

Reply:
Sure.